.github/actions/docker/action.yaml

```diff
@@ -77,18 +76,20 @@ runs:
         set -x
         DOCKER_COMPOSE_IMAGE="${GITHUB_DOCKER_REGISTRY_URL}/${{ github.repository }}/${DOCKER_SERVICE}"
 
-        if docker manifest inspect "${DOCKER_COMPOSE_IMAGE}:${DOCKER_CONTENT_SHA}"; then
-          # A postsubmit image exists, use it.
-          echo "Using existing ${DOCKER_COMPOSE_IMAGE}"
+        if [[ "$IS_PR" == "true" ]]; then
+          if docker manifest inspect "${DOCKER_COMPOSE_IMAGE}:${DOCKER_CONTENT_SHA}"; then
+            # A postsubmit image exists, use it.
+            echo "Using existing ${DOCKER_COMPOSE_IMAGE}"
 
-        else
-          # No fitting post-submit image was found. Check PR registry to see if there's an image there.
-          PROJECT_NAME=$(gcloud config get-value project)
-          DOCKER_COMPOSE_IMAGE="${PR_DOCKER_REGISTRY_URL}/${PROJECT_NAME}/github/${DOCKER_SERVICE}"
+          else
+            # No fitting post-submit image was found. Check PR registry to see if there's an image there.
+            PROJECT_NAME=$(gcloud config get-value project)
+            DOCKER_COMPOSE_IMAGE="${PR_DOCKER_REGISTRY_URL}/${PROJECT_NAME}/github/${DOCKER_SERVICE}"
 
-          if ! docker manifest inspect "${DOCKER_COMPOSE_IMAGE}:${DOCKER_CONTENT_SHA}"; then
-            echo "::warning::No pre-built docker image found for tag ${DOCKER_CONTENT_SHA}."
-            echo "build_image=true" >> $GITHUB_OUTPUT
+            if ! docker manifest inspect "${DOCKER_COMPOSE_IMAGE}:${DOCKER_CONTENT_SHA}"; then
```

Comment:
Why do we need to run `docker manifest inspect` again, we already do that on line 80?